src/php/.gitignore

```diff
@@ -18,5 +18,4 @@ missing
 mkinstalldirs
 
 ext/grpc/ltmain.sh
-composer.lock
```

Comment:
I am afraid we can't just check in composer.lock like this because that could break developers that check out grpc repo and want to work on PHP.
In a similar nature, we could actually try to get composer.lock created and all dependencies downloaded in the base PHP docker file. After that, we need to make sure the contents of the PHP repository would get wiped out properly. Then, when actually building PHP from the latest sources, composer should be able to recognize that all the sources have already been downloaded.